packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -566,3 +859,57 @@ class ReusePipelineStack extends cdk.Stack {
     });
   }
 }
+
+const createPipeline = (scope: Construct, pipelineName?: string, pipelineId: string = 'Pipeline') => {
```

Comment:
Also, can you put this function below `PipelineStack`, where it's used? It's much easier to read a file if it reads top to bottom.

Reply:
moved